files/chef-marketplace-gem/lib/marketplace/reckoner/metrics/erchef_crashes.rb

```diff
@@ -0,0 +1,17 @@
+require 'marketplace/reckoner/metrics/base'
+
+class Marketplace::Reckoner::Metrics
+  class ErchefCrashes < LogParser
+    def filenames
+      [
+        '/var/log/opscode/opscode-erchef/crash.log',
```

Comment:
@jamesc check out https://github.com/chef-partners/omnibus-marketplace/commit/5d84dd20db88223071822a9e4f19006b86da0d36 and see what you think. Should address the log-rolling concerns, I think, without a ton of added complexity.